packages/nuxt-vitest/src/config.ts

```diff
@@ -177,6 +177,14 @@ declare module 'vitest' {
        * @default {happy-dom}
        */
       domEnvironment?: 'happy-dom' | 'jsdom'
+      /**
+       * If true, polyfill the fetch.
+       *
+       * This is useful if native fetch is not available, such as for versions under Node18.
+       * 
+       * @default {false}
+       */
+      isFetchPolyfill?: boolean
```

Comment:
Rather than requiring an option, what about if we just polyfill fetch if `globalThis.fetch` is not set?

Reply:
Thanks for having a look at the code!
Of course it is better that way!
The reason I used this is that I wanted to show that $fetch would result in an error if there was no polyfill.
I'll simply modify it so that if there is no globalThs.fetch, it will polyfill.

Reply:
Ahh, of course. I understand.